### PR TITLE
add assests and landing component

### PR DESCRIPTION
**Pull request by shivamSspirit**
add asset css and landing page upper part components

### CODE REVIEW

File: src/components/land/land.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    position: relative;
+}
+
+.img-content {
+    display: flex;
+    align-items: center;
+    position: absolute;
+    top: 7.7rem;
+    max-width: 25rem;
+    /* text-align: left; */
```
Comment (HeyNitin) [inline src/components/land/land.css:24]: remove this

**Review by HeyNitin — APPROVED** [review]
remove commented code
rest all good, you can merge this

File: src/components/land/land.css (hunk 1 above)
Comment (Megha-Joshi) [reply src/components/land/land.css:24]: +1

File: src/components/land/land.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    padding: 1.5rem;
+    color: aliceblue;
```
Comment (Megha-Joshi) [inline src/components/land/land.css:26]: Use variable name for giving color.

**Review by Megha-Joshi — APPROVED** [review]
- Nice folder structuring
- Use CSS variables
- Remove the unused code
- Good to merge

**Pull request merged**